compose/install.md

```diff
@@ -95,185 +92,121 @@ on Microsoft Windows Server and want to install Docker Compose.
     ```
 
     Then run the following command to download the current stable release of
-    Compose (v{{site.compose_version}}):
+    Compose ({{site.compose_version}}):
 
     ```powershell
     Invoke-WebRequest "https://github.com/docker/compose/releases/download/{{site.compose_version}}/docker-compose-Windows-x86_64.exe" -UseBasicParsing -OutFile $Env:ProgramFiles\Docker\docker-compose.exe
     ```
 
-**Note**: On Windows Server 2019, you can add the Compose executable to `$Env:ProgramFiles\Docker`. Because this directory is  registered in the system `PATH`, you can run the `docker-compose --version` command on the subsequent step with no additional configuration.
+    >**Note**:  
+    > On Windows Server 2019, you can add the Compose executable to `$Env:ProgramFiles\Docker`. Because this directory is  registered in the system `PATH`, you can run the `docker-compose --version` command on the subsequent step with no additional configuration.
 
-    > To install a different version of Compose, substitute `{{site.compose_version}}`
+    > To install a different version of Compose, substitute `{{site.compose_version}}` 
```

Comment:
I think the trailing whitespace may cause a line break to be forced; don't think that was intentional, so better to remove it (also see above)

```suggestion
    > To install a different version of Compose, substitute `{{site.compose_version}}`
```